src/main/java/homework_2/pyramid_printer/Main.java

```diff
@@ -0,0 +1,11 @@
+package homework_2.pyramid_printer;
+
+import homework_2.utils.Executable;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Executable pyramidPrinter = new PyramidPrinter();
+        pyramidPrinter.run();
```

Comment:
Opt: could be simplified to 
```suggestion
        new PyramidPrinter().run();
```